lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -10,6 +10,21 @@ include_once('dosomething_helpers.theme.inc');
 include_once('dosomething_helpers.strongarm.inc');
 include_once('dosomething_helpers.variable.inc');
 
+
+/**
+ * Utility function to check if a variable is set, and return it
+ * if it is, otherwise return a default value.
+ *
+ * @param $var
+ * @param null $default
+ *
+ * @return array|object|string|integer|null
+ */
+function dosomething_helpers_issetor(&$var, $default = NULL) {
```

Comment:
Minor, but this name just feels weird to me.. like its a mispelling of `setter`. Also is passing the variable by reference on purpose?